content/en/profiler/enabling/ebpf.md

```diff
@@ -0,0 +1,134 @@
+      tag: 'Documentation'
+      text: 'Fix problems you encounter while using the profiler'
+---
+
+<div class="alert alert-warning">
+<code>ebpf</code> is in beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+The eBPF profiler is our eBPF-based host profiler built on OpenTelemetry and designed to send profiling data to the Datadog backend via the Datadog Agent. It supports symbolication for compiled languages and is optimized for containerized environments like Docker and Kubernetes.
+
+## Requirements
+
+For a summary of the minimum and recommended runtime and tracer versions across all languages, read [Supported Language and Tracer Versions][7].
+
+Supported operating systems
+: Linux (glibc or musl)
+
+Supported architecture
+: `amd64` or `arm64` processors
+
+Serverless
+: `ebpf` is not supported on serverless platforms, such as AWS Lambda.
+
+OS Settings
+: `perf_event_paranoid` kernel setting is 2 or less (see [Troubleshooting][1])
+
+Debugging information
+: Symbols should be available. The profiler cannot provide human-readable function names if the symbol table is stripped.
+
+## Installation
+
+The profiler can be used either as a standalone executable or as a library. Skip to [library installation instructions](#library) if you want to use it as a library.
+
+### Running the Profiler
+If the host is running workloads inside containers, it is recommended to run the profiler inside a container as well. A container image is [available][5].
+
+If you're using Kubernetes, follow the instruction for [running in Kubernetes][8].
+
+If you're directly using Docker, follow the instrcutins for [running in Docker][9].
+
+If you're not using a container runtime, follow the instructions for running [directly on the host][10]
+
+### Configuring the Profiler
+#### Local symbol upload (Experimental)
+For compiled languages (C/C++/Rust/Go/...), the profiler can upload local symbols (when available) to Datadog for symbolication. Symbols need to be available locally (unstripped binaries).
+
+To enable local symbol upload:
+
+1. Set the `DD_HOST_PROFILING_EXPERIMENTAL_UPLOAD_SYMBOLS` environment variable to true.
+2. Provide a Datadog API key through the `DD_API_KEY` environment variable.
+3. Provide a Datadog APP key through the `DD_APP_KEY` environment variable.
+4. Set the `DD_SITE environment` variable to your Datadog site (e.g. datadoghq.com, datadoghq.eu, us5.datadoghq.com, ...).
+
+### Build
+You must first ensure you have the correct version of go installed. In order to build the profiler directly on your machine, you can simply run:
+
+   ```shell
+   make
+   ```
+
+### Library
+TBD
+
+### Deploying the shared library
+
+TBD
+
+#### Installing the library
+
+TBD
+#### Appending a search directory
+
+TBD
+
+## Configuration
+
+The `environment`, `service`, and `service_version` settings are recommended, as they are used by the Profiling UI.
```

Comment:
We might want to put this as TBD as well, until we've figured out the right configuration bits (e.g. with the current planned work, `service` is likely not relevant)